modules/manually-maintained-credentials-upgrade.adoc

```diff
@@ -5,9 +5,28 @@
 // * installing/installing_gcp/manually-creating-iam-gcp.adoc
 
 [id="manually-maintained-credentials-upgrade_{context}"]
-== Upgrades
+= Upgrading clusters with manually maintained credentials
 
-In a future release, improvements to the Cloud Credential Operator will prevent
-situations where a user might enter an upgrade that will fail because their
-manually maintained credentials have not been updated to match the
-`CredentialsRequests` in the upcoming release image.
+If permissions change in a future release, the Cloud Credential Operator (CCO)
+`upgradable` status for a cluster with manually maintained credentials changes
+to `false`. For minor releases (for example, from 4.5 to 4.6), this status
+prevents you from upgrading until you have addressed any updated permissions.
```

Comment:
Possible nit here, this forces you to create Secrets for any new credentials. We don't/can't actually check yet for new permissions being added or removed from existing credentials. We might be able to improve this in the future but it could be tricky.

Reply:
I added lines 25-29 to cover the new permissions for existing creds scenario. So would it work to change the first line here as follows and then leave the rest of the section as-is?? 
`If credentials are added in a future release,`

Reply:
:+1: